src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -54,20 +78,17 @@ public CaseSearchEndpoint(@Qualifier(AuthorisedCaseSearchOperation.QUALIFIER) Ca
     public CaseSearchResult searchCases(
         @ApiParam(value = "Case type ID(s)", required = true)
         @RequestParam("ctid") List<String> caseTypeIds,
-        @ApiParam(value = "Native ElasticSearch Search API request. Please refer to the ElasticSearch official "
-            + "documentation. For cross case type search, "
-            + "the search results will contain only metadata by default (no case field data). To get case data in the "
-            + "search results, please state the alias fields to be returned in the _source property for e.g."
```

Comment:
replaced the wrong @ApiParam. Please put this back and replace the one at line 79 i.e. 
``` 
@ApiParam(value = "Case type ID(s)", required = true)